### PR TITLE
feat: new sample - list all compute instances

### PR DESCRIPTION
**Pull request by rsamborski**
Adding GCE sample to list all instances for particular projectId

### CODE REVIEW

**Comment by rsamborski** [comment]
To be merged after #1353

File: compute/cloud-client/instances/src/list_all_instances.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+ *
+ * @param string $projectId Your Google Cloud project ID.
+ *
+ * @throws \Google\ApiCore\ApiException if the remote call fails.
+ */
+function list_all_instances(string $projectId)
+{
+    // List the new Compute Engine instance using the InstancesClient
+    $instancesClient = new InstancesClient();
+    $aggList = $instancesClient->aggregatedList($projectId);
```
Comment (bshaffer) [inline compute/cloud-client/instances/src/list_all_instances.php:44]: Let's just call this variable `$allInstances`, I think that name is more descriptive of what it actually is
```suggestion
    $allInstances = $instancesClient->aggregatedList($projectId);
```

File: compute/cloud-client/instances/src/list_all_instances.php [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+    printf('All instances for %s' . PHP_EOL, $projectId);
+    foreach ($aggList as $zone => $response) {
```
Comment (bshaffer) [inline compute/cloud-client/instances/src/list_all_instances.php:47]: Let's call this `$zoneInstances`, as that's more descriptive of what this is
```suggestion
    foreach ($allInstances as $zone => $zoneInstances) {
```

**Comment by rsamborski** [comment]
@bshaffer @ace-n can you please take a look at this PR? This sample is going to be part of new library release.

**Review by bshaffer — APPROVED** [review]

**Pull request merged**